src/status_im/contexts/wallet/add_account/create_account/import_private_key/style.cljs

```diff
@@ -0,0 +1,36 @@
+(ns status-im.contexts.wallet.add-account.create-account.import-private-key.style
+  (:require [quo.foundations.colors :as colors]))
+
+(def input
+  {:margin-top         12
+   :padding-horizontal 20})
+
+(def indicator
+  {:padding-horizontal 20
+   :padding-vertical   8})
+
+(def info-box
+  {:margin-bottom 20})
+
+(def page-top
+  {:margin-top 2})
+
+(def key-section
+  {:margin-top         22
+   :padding-horizontal 20})
+
+(def section-label
+  {:margin-bottom 8})
+
+(defn public-address
+  [state]
+  (let [border-color (case state
+                       :active-address   colors/success-50-opa-40
```

Comment:
nit, probably should be themed too.

`:active-address (colors/resolve-color :success  theme 40)` - will be right 👍 
`:inactive-address  (colors/resolve-color :warning  theme 40)`